src/filter-images.js

```diff
@@ -0,0 +1,13 @@
+function filterImages(images, filter) {
+    const lowerCaseTitleFilter = filter.title.toLowerCase();
+    return images.filter(image => {
+        const lowerCaseTitle = image.title.toLowerCase();
+
+        const hasTitle = !lowerCaseTitleFilter || lowerCaseTitle.includes(lowerCaseTitleFilter);
+        const hasHorns = !filter.horns || image.horns === filter.horns;
+
+        return hasTitle && hasHorns;
```

Comment:
Right now you are getting 1 point off because you are only checking the filter text against 1 property on the image object. 
You could repeat the same type of logic that you used for filtering against the title and line 9 could look something like 
`return (hasTitle || hasDescription || hasKeywords) && hasHorns`

Reply:
Additionally the function might look something like this if you wanted to check more than 1 property on the image.

`unction filterImages(images, filter) {
    const lowerCaseTitleFilter = filter.title.toLowerCase();
    return images.filter(image => {
        const lowerCaseTitle = image.title.toLowerCase();
        const lowerCaseDescription = image.description.toLowerCase();

        const hasText = !lowerCaseTitleFilter || lowerCaseTitle.includes(lowerCaseTitleFilter) || lowerCaseDescription.includes(lowerCaseTitleFilter);


        const hasHorns = !filter.horns || image.horns === filter.horns;

        return hasText && hasHorns;
    });
}

export default filterImages;`